fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1776,6 +1776,12 @@ public class Config extends ConfigBase {
             "Maximum number of persistence allowed per task in a job,exceeding which old tasks will be discarded，"
                    + "If the value is less than 1, it will not be persisted." })
     public static int max_persistence_task_count = 100;
+    @ConfField(description = {"MV task 的等待队列大小，值必须是 2 的幂，否则将使用默认值 1024", "The size of the MV task's"
```

Comment:
use `setter` to check if value is power of 2.
And you can also check if the value is too large or too small.
And need to test unit test